tests/fixups/TraceFixup/CreateProcessFixup.cpp

```diff
@@ -26,13 +26,51 @@ BOOL __stdcall CreateProcessFixup(
     _In_ startupinfo_t<CharT> startupInfo,
     _Out_ LPPROCESS_INFORMATION processInformation)
 {
+	LARGE_INTEGER TickStart, TickEnd;
+	QueryPerformanceCounter(&TickStart);
     auto entry = LogFunctionEntry();
     auto result = CreateProcessImpl(applicationName, commandLine, processAttributes, threadAttributes, inheritHandles, creationFlags, environment, currentDirectory, startupInfo, processInformation);
-    preserve_last_error preserveError;
+	preserve_last_error preserveError;
+	QueryPerformanceCounter(&TickEnd);
 
     auto functionResult = from_win32_bool(result);
     if (auto lock = acquire_output_lock(function_type::process_and_thread, functionResult))
     {
+		if (output_method == trace_method::eventlog)
+		{

```

Comment:
along with consolidation comment in Logging.h, it looks like much of the call sites could also be abstracted and consolidated.  Should be able to avoid doubling code size.